src/app/_mermaid_common/libs/services/offline.svc.js

```diff
@@ -219,16 +219,16 @@ angular.module('mermaid.libs').service('offlineservice', [
           }
         }
 
-        return ProjectsTable(null, true).then(function(table) {
+        return ProjectsTable(true).then(function(table) {
           return table.get(projectId).then(function(record) {
             return record !== null;
           });
         });
       });
     };
 
-    const getOfflineProjects = function(profileId) {
-      return ProjectsTable(profileId, true)
+    const getOfflineProjects = function() {
```

Comment:
I think the only place this is used is in `projects.ctrl.js` but that place isn't updated to remove the arg, unless I'm missing it